utils/resource-monitor/README.md

```diff
@@ -0,0 +1,36 @@
+# Resource performance monitoring and graphing module
+
+This is a module for monitoring the resource usage of batch mode with HAPI as the source. The scripts in this module generate graphs and csv files of resource usage (CPU, memory and I/O) of the HAPI server, postgres database and pipeline over the duration of the batch job. The user is able to specify the number of processes/cores used in the pipeline to assess batch mode's performance on the local machine.
```

Comment:
Please break long lines to at most 100 chars; in general take a look at the [style guide](https://google.github.io/styleguide/docguide/style.html) too.

Some other suggestions for this line:
- "batch mode" -> "batch pipeline"
- add a link to HAPI